app/views/layouts/_patientnav.html.slim

```diff
@@ -0,0 +1,52 @@
+nav.navbar.navbav-default#patient-nav
+  .container
+    .row
+      .col-md-3
+        h4 = "#{@patient.first_name} #{@patient.last_name}"
+        ul.nav.nav-pills 
+          li 
+            | SSN:
+            = @patient.ssn
+      .col-md-3
+        ul.nav.nav-pills 
+          li.top
+            | DOB: 
+            = @patient.dob
+          li
+            | SCI Identification:
+            = @patient.patient_id
+      .col-md-3
+        ul.nav.nav-pills 
+          li.top
```

Comment:
This is gonna need to be rebased or merged on top of the patient registration changes since some of these fields have gone away.